src/components/CallToAction/CallToAction.css

```diff
@@ -0,0 +1,38 @@
+.callToAction {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 1219px;
+  background: #343434;
+  border-radius: 33px;
+  margin: auto;
+  margin-top: 60px;
```

Comment:
- [ ] use relative value for width
- [ ] use round /even figure for border-radius